src/main/java/utils/Constants.java

```diff
@@ -32,5 +32,7 @@ public class Constants {
     public static final String INVOICE_DATE = "07/07/2022";
     public static final String name = randomName() + "";
     public static final String number = "0000" + randomNumber() + "";
+    public static final String TAG_NAME_WS = "whitepaces ";
```

Comment:
can't we reuse the WHITE_SPACE_NAME_WS constant here?

Reply:
True